sw/device/tests/pmod/BUILD

```diff
@@ -65,6 +64,7 @@ opentitan_test(
     ),
     fpga = fpga_params(
         tags = [
+            "cw310_sival_rom_ext",
```

Comment:
If I'm not misunderstanding, would doing this not make the `cw310_sival_rom_ext` tag appear on all tests that use `fpga`, including e.g. `fpga_cw310_test_rom`? If so, would it not be better to make a separate `fpga_params` entry for the `fpga_cw310_sival_rom_ext_no_hyper` environment?